src/KafkaFlow.Admin.Dashboard/KafkaFlow.Admin.Dashboard.csproj

```diff
@@ -23,7 +23,6 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="6.0.20" />
```

Comment:
I think @filipeesch can shed some light here, if this is necessary or not.

Reply:
https://github.com/Farfetch/kafkaflow/pull/486#discussion_r1422258927